spotlight-client/src/SiteNavigation/SiteNavigation.tsx

```diff
@@ -89,7 +91,7 @@ const SiteNavigation: React.FC<ShareButtonProps> = ({ openShareModal }) => {
                   params: { tenantId: tenant.id },
                 })}
               >
-                {tenant.name}
+                {isTablet ? tenant.name : tenant.docName}
```

Comment:
So we want to hide the docName if our width is tablet size? What about abbreviating the docName like "Pennsylvania DOC"?

Reply:
<img width="770" alt="Screen Shot 2022-04-25 at 9 53 38 AM" src="https://user-images.githubusercontent.com/5040739/165137399-ff305c1c-af00-491c-8f3f-94d244c47e26.png">
Are we trying to avoid two lines of text like this?

Reply:
Not sure what you mean by this not happens?

Reply:
I mean the two lines case not happens, when we load page on small screens. 

I saw this happening only when resizing window in DevTools.